.gitmodules

```diff
@@ -9,7 +9,7 @@
 	url = https://github.com/weidai11/cryptopp.git
 [submodule "lmdb"]
 	path = lmdb
-	url = https://github.com/clemahieu/lmdb.git
+	url = https://github.com/arugulaBowl/lmdb.git
```

Comment:
Seems like same LMDB (?)

Reply:
Are you looking at the right branch?

Reply:
clemahieu fork replaced with arugulaBowl/lmdb

Reply:
Nevermind... yeah I don't see a difference.

Reply:
The only change to lmdb here is the definition of NTSTATUS which shouldn't be required unless targeting old MinGW versions - is that the case @arugulaBowl ?

Reply:
Ah yes, I didn't explain that well. This is the same as clemahieu/lmdb_0_9_21 with this extra commit on top: https://github.com/arugulaBowl/lmdb/commit/58d12fb3abd8d50b7a81d435c6409bb1bc1af9f5.

I don't understand why this was needed now though.